usr/lib/python3/dist-packages/linuxmusterLinuxclient7/printers.py

```diff
@@ -113,12 +117,17 @@ def _getInstalledPrintersOfUser(username):
     return True, installedPrinters
 
 def _uninstallPrinter(name):
-    logging.info("* Uninstalling Printer {}".format(name))
-    uninstallCommand = ["lpadmin", "-x", name]
-    #logging.debug("* running '{}'".format(installCommand))
-
-    if not subprocess.call(uninstallCommand) == 0:
-        logging.error("* Error uninstalling printer!")
-        return False
-
-    return True
+    logging.info("Uninstall Printer {}".format(name))
+    uninstallCommand = ["timeout", "10", "lpadmin", "-x", name]
+    logging.debug("* running '{}'".format(" ".join(uninstallCommand)))
+    p = subprocess.Popen(uninstallCommand, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    output, error = p.communicate()
+    if p.returncode  == 0:
+        logging.debug("* Success Uninstall Printer!")
+        return True
+    elif p.returncode == 124:
+        logging.debug("* Error timeout lpadmin")
+    else:
+        logging.debug(f"* Error Uninstall Printer {name}!")
```

Comment:
```suggestion
        logging.fatal(f"* Error Uninstall Printer {name}!")
```
This should be fatal.